pkg/cimfs/cim_writer_windows.go

```diff
@@ -173,21 +234,41 @@ func (c *CimFsWriter) AddLink(oldPath string, newPath string) error {
 	return err
 }
 
-// Unlink deletes the file at `path` from the image.
+// AddMergedLink adds a hard link from `oldPath` to `newPath` in the image. However unlike
```

Comment:
This seems backward. Shouldn't links be newPath -> oldPath?

Reply:
This still seems wrong?

Reply:
Fixed now. My bad.